BUILD.bazel

```diff
@@ -95,7 +97,7 @@ GA_LIBRARIES = GOOGLE_CLOUD_CPP_GA_LIBRARIES
             library_dir = google_cloud_cpp_library_dir_name(library),
         ),
     ],
-) for library in GA_LIBRARIES + TRANSITION_LIBRARIES]
+) for library in GA_LIBRARIES + TRANSITION_LIBRARIES if library not in NO_MOCK_LIBRARIES]
```

Comment:
We should probably add this to the `experimental-{library}_mocks` too.

Reply:
Done.